### ISSUE TITLE
Exception in plugin TLA+ (0.9.4) 

### ISSUE THREAD

**Issue by lemmy** [ocadaruma/tlaplus-intellij-plugin#40]
```
IntelliJ IDEA 2024.2.1 (Community Edition)
Build #IC-242.21829.142, built on August 28, 2024
Runtime version: 21.0.3+13-b509.11 aarch64 (JCEF 122.1.9)
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Toolkit: sun.lwawt.macosx.LWCToolkit
macOS 13.6.9
GC: G1 Young Generation, G1 Concurrent GC, G1 Old Generation
Memory: 4096M
Cores: 10
Metal Rendering is ON
Registry:
  ide.experimental.ui=true
  i18n.locale=
  scala.erase.compiler.process.jdk.once=false
Non-Bundled Plugins:
  com.mayreh.tlaplus-intellij-plugin (0.9.4)
  org.jetbrains.plugins.gitlab (242.21829.162)
  com.github.copilot (1.5.20.6554)
  org.intellij.scala (2024.2.25)
Kotlin: 242.21829.142-IJ
```

```scala
com.intellij.diagnostic.PluginException: `ActionUpdateThread.OLD_EDT` is deprecated and going to be removed soon. 'com.mayreh.intellij.plugin.tlaplus.ide.actions.PlusCalTranslateAction' must override `getActionUpdateThread()` and chose EDT or BGT. See ActionUpdateThread javadoc. [Plugin: com.mayreh.tlaplus-intellij-plugin]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:90)
	at com.intellij.diagnostic.PluginException.reportDeprecatedUsage(PluginException.java:125)
	at com.intellij.openapi.actionSystem.ActionUpdateThreadAware.getActionUpdateThread(ActionUpdateThreadAware.java:21)
	at com.intellij.openapi.actionSystem.AnAction.getActionUpdateThread(AnAction.java:201)
```

**Comment by ocadaruma** [comment]
Thank you for the report.
Let me check

**Comment by ocadaruma** [comment]
@lemmy The fixed version is released as 0.9.5.
